### PR TITLE
Improve transistor definitions in xschem lvs export

### LINKED ISSUE #101: lvs script should be able to import xschem netlist
By default xschem extract netlist with the `.spice` extension and include modeling equation as part of the parameters (which klayout can't parse).

It would be nice if the lvs script was trying multiple extension and simplifying the netlist prior to parsing.

### PR DESCRIPTION
**Pull request by StefanSchippers**
- strip unusted parameters on LVS export
- avoids X prefix
- fix substrate pin of vnpn_10x10.sym (was named "B" instead of "S")

 (Still to be decided if we want to make similar changes for LVS on other devices (moscap and others)

Fixes #101

### CODE REVIEW

File: cells/xschem/symbols/vnpn_10x10.sym [hunk 1]
```diff
@@ -33,7 +33,7 @@ L 4 0 -10 20 -30 {}
 B 5 17.5 -32.5 22.5 -27.5 {name=C dir=inout pinnumber=3}
 B 5 -22.5 -2.5 -17.5 2.5 {name=B dir=in pinnumber=1}
 B 5 17.5 27.5 22.5 32.5 {name=E dir=inout pinnumber=2}
-B 5 17.5 -2.5 22.5 2.5 {name=B dir=in pinnumber=1}
+B 5 17.5 -2.5 22.5 2.5 {name=S dir=in pinnumber=1}
```
Comment (proppy) [inline cells/xschem/symbols/vnpn_10x10.sym:36]: @StefanSchippers is this change related?

**Review by proppy — APPROVED** [review]
Just a nit.

File: cells/xschem/symbols/vnpn_10x10.sym (hunk 1 above)
Comment (StefanSchippers) [reply cells/xschem/symbols/vnpn_10x10.sym:36]: The latter one fixes a wrong pin in the vnpn transistor. It has the substrate pin incorrectly named as the base (B)  pin, making base and substrate shorted.
The fix renamed the substrate pin from 'B' to 'S'

Comment (proppy) [reply cells/xschem/symbols/vnpn_10x10.sym:36]: Make sense, updating the PR so that it makes it clearer that it fixes both before merging.

**Comment by atorkmabrains** [comment]
@StefanSchippers Could you please make sure to add a similar PR on: https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pr

**Comment by proppy** [comment]
> @StefanSchippers Could you please make sure to add a similar PR on: https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pr

Done here: https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/23

**Comment by proppy** [comment]
@StefanSchippers can `lvs_format` also be used to other component used only by simulation from the LVS netlist? (`C` blocks for lab pin, spice model, etc)

**Comment by StefanSchippers** [comment]
If you add an `lvs_format` this takes precedence over `format` when doing LVS netlists.
This can be used to exclude simulation-only devices from LVS netlist, like for example in:
`lvs_format="* @name @pinlist @value"`
Notice the '`*`' at the beginning. This will add just a commented line in the netlist.
Just make sure lvs_format is not empty, because in this case the xschem netlister will revert to format attribute. Even a single space will be ok, like in:
`lvs_format=" "`

**Comment by proppy** [comment]
> Just make sure lvs_format is not empty, because in this case the xschem netlister will revert to format attribute. Even a single space will be ok, like in:
lvs_format=" "

Yes, I went for `lvs_format="*"` :) Thanks for the tip!
